modules/compiler/spirv-ll/source/builder_core.cpp

```diff
@@ -621,12 +621,19 @@ cargo::optional<Error> Builder::create<OpTypeEvent>(const OpTypeEvent *op) {
   return cargo::nullopt;
 }
 
+static Error errorUnsupportedDeviceEnqueueOp(const std::string &opName) {
+  // Capability DeviceEnqueue isn't supported by CL 1.2, see OpenCL SPIR-V
+  // environment spec section 3.1 for supported capabilities.
+  // It is, however, implicitly declared by SPIR-V 1.1 modules which declare the
+  // SubgroupDispatch capability for CL 3.0 devices supporting subgroups. This
+  // is highly dubious, and appears like a spec bug of some kind.
```

Comment:
Should there be a jira to address this?

Reply:
I've created an internal ticket to look into this.